src/lib/SpotifyAPI.ts

```diff
@@ -28,28 +29,25 @@ export class SpotifyAPI {
 
   playlist: PlaylistManager;
 
-  private privateConfig: PrivateConfig = {};
-
-  config: SpotifyConfig;
-
   constructor(config: SpotifyConfig) {
-    this.config = config;
```

Comment:
I know of multiple projects that are using the `<SpotifyApi>.config` to retrieve the configuration options and update the tokens through there, due to the removal of this line of code it's not possible anymore. I would advice to set it as public in the constructor directly, saves some lines and then it isn't a breaking change for other projects that are making use of this.

Reply:
Thank you for sharing @stijnvdkolk, will improve the solution according to this fact. Maybe we have anything else details which may make sense for the library clients?